tests/integration/_validator/testcases/test_validator.go

```diff
@@ -129,30 +130,46 @@ func (t *ValidatorTest) testInstallInteractive(ctx *test.TestContext) (tr *test.
 	return common.ExecCLI(interactiveCmd, buffer, t.log, false)
 }
 
-func (t *ValidatorTest) testInstallInteractiveCheck(ctx *test.TestContext) (tr *test.TestResult) {
-	t.log.Printf("Executing testInstallInteractiveCheck")
+func (t *ValidatorTest) testInstallInteractiveApplyExistingCluster(ctx *test.TestContext) (tr *test.TestResult) {
+	t.log.Printf("Executing testInstallInteractiveApply")
 
-	interactiveCmd, buffer := common.InitCmd([]string{"install", "-o", "--apply", "-l", "debug"})
+	kindClusterName = fmt.Sprintf("%s-%s", cfg.ValidatorKindClusterName, string_utils.RandStr(5))
+	kindConfig := t.filePath("kindconfig.yaml")
+	kubeConfig := t.filePath("kubeconfig.yaml")
+
+	if err := kind.StartCluster(kindClusterName, kindConfig, kubeConfig); err != nil {
+		t.log.Errorf("Failed to start validator kind cluster: %v", err)
+	}
+	err := os.Setenv("KUBECONFIG", kubeConfig)
+	if err != nil {
+		t.log.Errorf("Failed to set KUBECONFIG: %v", err)
+	}
+
+	interactiveCmd, buffer := common.InitCmd([]string{"install", "--apply", "-l", "debug"})
+	preProvisionedKindCluster := true
 
 	// Base values
-	tuiVals := t.validatorValues(ctx)
+	tuiVals := t.validatorValues(ctx, preProvisionedKindCluster)
 
 	// Install values
-	tuiVals = t.awsPluginInstallValues(ctx, tuiVals)
-	tuiVals = t.azurePluginInstallValues(ctx, tuiVals)
-	tuiVals = t.maasPluginInstallValues(ctx, tuiVals)
+	tuiVals = t.awsPluginInstallValues(ctx, tuiVals, preProvisionedKindCluster)
+	tuiVals = t.azurePluginInstallValues(ctx, tuiVals, preProvisionedKindCluster)
+
+	// TODO: re-enable mass plugin once its TUI prompts are fixed
```

Comment:
Given that these tests werent really testing the plugin install values, its not a surprise that the maas tui prompts arent configured correctly.

for now ive opted to comment the maas plugin out of the integration tests, but would love it if @arturshadnik can take a stab at fixing them later.

Reply:
@ahmad-ibra this test was never intended to test the values. It purely tests the user input flow. The silent tests are for testing the plugin values.

Reply:
re-enabled it and will rebase my PR once the fix is in place